docs: Create Contribution Guidelines (#49)
ScottBrenner/cfn-lint-action: CONTRIBUTING.md

## Contributing

> This product welcomes contributions the any open source community. Firstly, please raise an issue on the repository, as this will be used to discuss/agree on scope and feature/bug detail. Once agreed, please follow the [Running Locally](#running-locally) steps above to get started on your machine. Branch off `main` following logically branch naming conventions `feature/***` or `bug/***`. When contributing please try and meet these development principles:

Author: ```suggestion
This product welcomes contributions the any open source community. Firstly, please raise an issue on the repository, as this will be used to discuss/agree on scope and feature/bug detail. Once agreed, please follow the Development section's steps in [README.md](README.md) to get started. Branch off `main` following logically branch naming conventions `feature/***` or `bug/***`. When contributing please try and meet these development principles:
```
